app/src/main/java/org/openobservatory/ooniprobe/test/suite/CircumventionSuite.java

```diff
@@ -39,9 +39,9 @@ public static CircumventionSuite initForAutoRun() {
     @Override public AbstractTest[] getTestList(@Nullable PreferenceManager pm) {
         if (super.getTestList(pm) == null) {
             ArrayList<AbstractTest> list = new ArrayList<>();
-            if (pm == null || pm.isTestPsiphon())
+            // if (pm == null || pm.isTestPsiphon())
                 list.add(new Psiphon());
-            if (pm == null || pm.isTestTor())
+            // if (pm == null || pm.isTestTor())
```

Comment:
if these commented out lines are no longer needed, I'd zap them

Reply:
This comment is still here since the strategy for automated test wasn't too clear at the time.Once the functionality is implemented , these classes would be cleaned up